VimeoNetworking/Sources/Request.swift

```diff
@@ -153,17 +153,17 @@ public struct Request<ModelType: MappableResponse>
         self.path = path
         self.parameters = parameters
         self.modelKeyPath = modelKeyPath
-        self.cacheFetchPolicy = cacheFetchPolicy ?? CacheFetchPolicy.defaultPolicyForMethod(method)
+        self.cacheFetchPolicy = cacheFetchPolicy ?? CacheFetchPolicy.defaultPolicyForMethod(method: method)
         self.shouldCacheResponse = shouldCacheResponse ?? (method == .GET)
-        self.retryPolicy = retryPolicy ?? RetryPolicy.defaultPolicyForMethod(method)
+        self.retryPolicy = retryPolicy ?? RetryPolicy.defaultPolicyForMethod(method: method)
     }
     
         /// Returns a fully-formed URI comprised of the path plus a query string of any parameters
     public var URI: String
     {
         let URI = self.path
         
-        let components = NSURLComponents(string: URI)
+        var components = URLComponents(string: URI)
```

Comment:
We're assigning to its `query` property a little further down and it won't let you assign if it's not a var, which makes sense. I'm not sure why this wasn't a problem before though.